### PR TITLE
docs: add Cacti 2024 Q3 report

### PR DESCRIPTION
**Pull request by petermetz**
Signed-off-by: Peter Somogyvari <peter.somogyvari@accenture.com>

Edited the 2024 Q3 Cacti Report

Signed-off-by: VRamakrishna <vramakr2@in.ibm.com>

docs: update 2024 Q3 Cacti report

Signed-off-by: Rafael Belchior <rafael.belchior@tecnico.ulisboa.pt>

### CODE REVIEW

**Review by tkuhrt — APPROVED** [review]

**Review by swcurran — APPROVED** [review]

**Review by denyeart — APPROVED** [review]

**Review by mbrandenburger — APPROVED** [review]

File: project-reports/2024/2024-Q3-Hyperledger-Cacti.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+- Multiple mentorship projects are underway, namely [Enabling Enterprise Organizational Blockchain Interoperability](https://wiki.hyperledger.org/display/INTERN/Enabling+Enterprise+Organizational+Blockchain+Interoperability) and [Cacti: Core Operators Modules for DLTs](https://wiki.hyperledger.org/display/INTERN/Cacti%3A+Core+Operators+Modules+for+DLTs).
+- `v2.0.0-rc.2` was released a couple weeks ago and `v2.0.0-rc.3` is not far behind (hopefully next week)!
+- Major efforts are underway to integrate the codebase further
+- We continue to promote interoperability protocol standardization by contributing to, and implementing, the candidate [IETF SATP specification](https://datatracker.ietf.org/doc/draft-ietf-satp-core/) using different combinations of Cacti modules. We conducted a [workshop on May 23, 2024](https://wiki.hyperledger.org/display/events/Standardizing+DLT+Interoperation%3A+Implementing+IETF+Secure+Asset+Transfer+Protocol+in+Hyperledger+Cacti) to discuss the protocol and demonstrate the associated Cacti capabilities.
+- Enabled the [OpenSSF Scorecard](https://scorecard.dev/viewer/?uri=github.com%2Fhyperledger%2Fcacti) for the project.
```
Comment (arsulegai) [inline project-reports/2024/2024-Q3-Hyperledger-Cacti.md:16]: [comments]
1. +1
2. Does the project team have a plan to triage and/or mitigate the vulnerabilities called out in the scorecard?

File: project-reports/2024/2024-Q3-Hyperledger-Cacti.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+parent: 2024
+grand_parent: Project Updates
+---
+
+
+# Project Health
+
+- The project is in good health!
+- Multiple mentorship projects are underway, namely [Enabling Enterprise Organizational Blockchain Interoperability](https://wiki.hyperledger.org/display/INTERN/Enabling+Enterprise+Organizational+Blockchain+Interoperability) and [Cacti: Core Operators Modules for DLTs](https://wiki.hyperledger.org/display/INTERN/Cacti%3A+Core+Operators+Modules+for+DLTs).
+- `v2.0.0-rc.2` was released a couple weeks ago and `v2.0.0-rc.3` is not far behind (hopefully next week)!
```
Comment (arsulegai) [inline project-reports/2024/2024-Q3-Hyperledger-Cacti.md:13]:
```suggestion
- `v2.0.0-rc.2` was released a couple of weeks ago and `v2.0.0-rc.3` is not far behind (hopefully next week)!
```

**Review by arsulegai — APPROVED** [review]

File: project-reports/2024/2024-Q3-Hyperledger-Cacti.md (hunk 2 above)
Comment (petermetz) [reply project-reports/2024/2024-Q3-Hyperledger-Cacti.md:13]: @arsulegai Sorry for the extremely slow response! I would've been fine with accepting this suggestion of course, but now it's too late. Regardless, I wanted to come back and thank you for it and apologize again.

File: project-reports/2024/2024-Q3-Hyperledger-Cacti.md (hunk 1 above)
Comment (petermetz) [reply project-reports/2024/2024-Q3-Hyperledger-Cacti.md:16]: @arsulegai Yes, the biggest item we have is that for some reason it doesn't recognize the DAST we have in place through our GitHub actions (we use nuclei)

**Pull request merged**